BHoM_UI/Components/oM/CreateCustom.cs

```diff
@@ -79,9 +79,17 @@ public CreateCustomCaller() : base()
         /**** Public Methods              ****/
         /*************************************/
 
-        public void SetInputs(List<string> names)
+        public void SetInputs(List<string> names, List<Type> types = null)
         {
-            InputParams = names.Select(x => GetParam(x)).ToList();
+            if (names.Count != types.Count)
```

Comment:
if type is null (default value of the parameter), this will crash.

Reply:
Definitely, thanks for spotting it!